ghost/i18n/test/i18n-todos.json

```diff
@@ -0,0 +1,199 @@
+                "comment": "Auto-generated: addedVariable detected in translation"
+            },
+            {
+                "file": "da/portal.json",
+                "key": "Try free for {amount} days, then {originalPrice}.",
+                "comment": "Auto-generated: addedVariable detected in translation"
+            },
+            {
+                "file": "de-CH/portal.json",
+                "key": "Memberships unavailable, contact the owner for access.",
+                "comment": "Auto-generated: addedVariable detected in translation"
+            },
+            {
+                "file": "de-CH/portal.json",
+                "key": "We couldn't unsubscribe you as the email address was not found. Please contact the site owner.",
+                "comment": "Invalid variable added to translation"
+            },
+            {
+                "file": "is/portal.json",
+                "key": "Your subscription will renew on {renewalDate}",
+                "comment": "Auto-generated: addedVariable detected in translation"
+            },
+            {
+                "file": "it/portal.json",
+                "key": "Expires {expiryDate}",
+                "comment": "Auto-generated: addedVariable detected in translation"
+            },
+            {
+                "file": "lv/portal.json",
+                "key": "There was a problem submitting your feedback. Please try again a little later.",
+                "comment": "Auto-generated: addedVariable detected in translation"
+            },
+            {
+                "file": "si/portal.json",
+                "key": "Your subscription will renew on {renewalDate}",
+                "comment": "Auto-generated: addedVariable detected in translation"
+            },
+            {
+                "file": "si/portal.json",
+                "key": "Your subscription will start on {subscriptionStart}",
+                "comment": "Auto-generated: addedVariable detected in translation"
+            },
+            {
+                "file": "th/portal.json",
+                "key": "Try free for {amount} days, then {originalPrice}.",
+                "comment": "Auto-generated: addedVariable detected in translation"
+            }
+        ],
+        "missingVariable": [
+            {
+                "file": "ar/portal.json",
+                "key": "{amount} off",
+                "comment": "variable name error"
+            },
+            {
+                "file": "da/portal.json",
+                "key": "Try free for {amount} days, then {originalPrice}.",
+                "comment": "Auto-generated: missingVariable detected in translation"
+            },
+            {
+                "file": "de/portal.json",
+                "key": "Start {amount}-day free trial",
+                "comment": "translated text doesn't fit on the button",
+                "reference": "https://github.com/TryGhost/Ghost/pull/123456-fake-pr"
```

Comment:
🙌🏾

Reply:
Oops, I could probably tidy up the faked reference I was using for manual testing, huh?